core/templates/dev/head/components/create_button/CreateActivityButtonDirective.js

```diff
@@ -29,6 +29,7 @@ oppia.directive('createActivityButton', [
             CollectionCreationService, siteAnalyticsService, urlService) {
           $scope.creationInProgress = false;
 
+
```

Comment:
Please remove newline. In general, check the diff [here](https://github.com/oppia/oppia/pull/3724/files) when submitting and try not to introduce any extraneous elements in your PR.